entgql/annotation.go

```diff
@@ -70,6 +70,10 @@ type (
 	FieldConfig struct {
 		// Name is the name of the field in the Query object.
 		Name string `json:"Name,omitempty"`
+
+		// Description is the description of the field in the Query object.
```

Comment:
godoc is not correct.
```suggestion
		// Description is the description of the field.
```